haxelib/aze/display/SparrowTilesheet.hx

```diff
@@ -9,32 +9,41 @@ import flash.Lib;
 parrow spritesheet parser for TileLayer
  * - supports animations
  * - supports sprite trimming
- * - does NOT support sprite rotation
+ * - supports sprite rotation
  * @author Philippe / http://philippe.elsass.me
  */
 class SparrowTilesheet extends TilesheetEx
 {
 
-	public function new(img:BitmapData, xml:String, textureScale:Float = 1.0) 
+	public function new(img:BitmapData, xml:String, textureScale:Float = 1.0)
 	{
-		super(img, textureScale);
+		super(img, textureScale, -Math.PI/2);
 		
 		var ins = new Point(0,0);
 		var x = new haxe.xml.Fast( Xml.parse(xml).firstElement() );
+		var rotated:Bool = false;
 
 		for (texture in x.nodes.SubTexture)
 		{
 			var name = texture.att.name;
+			rotated = texture.has.rotated && texture.att.rotated == "true";
 			var rect = new Rectangle(
 				Std.parseFloat(texture.att.x), Std.parseFloat(texture.att.y),
 				Std.parseFloat(texture.att.width), Std.parseFloat(texture.att.height));
 			
-			var size = if (texture.has.frameX) // trimmed
-					new Rectangle(
-						Std.parseInt(texture.att.frameX), Std.parseInt(texture.att.frameY),
-						Std.parseInt(texture.att.frameWidth), Std.parseInt(texture.att.frameHeight));
-				else 
-					new Rectangle(0, 0, rect.width, rect.height);
+			var size = 	if (texture.has.frameX) { // trimmed

```

Comment:
Sloppy formatting - please follow the coding style (braces and `else` on new line).